dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -83,7 +83,7 @@ NULL_CHECKPOINT_VALUE=CWIML4550E: The LDAP operation could not be completed. An
 NULL_CHECKPOINT_VALUE.explanation=If a repository supports change tracking as specified by the 'supportChangeLog' flag for the repository in the server.xml file, then the checkpoint passed for it in the ChangeControl should not be empty or null.
 NULL_CHECKPOINT_VALUE.useraction=Try the search again with a valid checkpoint or disable 'supportChangeLog' for that repository in the server.xml file, if the user registry adapter is not capable of change tracking.
 
-MISSING_OR_EMPTY_PASSWORD=CWIML4541E: The login operation could not be completed. The password is missing or empty.
+MISSING_OR_EMPTY_PASSWORD=CWIML4541E: The login operation could not be completed because the password was either missing or empty.
```

Comment:
Reviewed Message: The login operation could not be completed as the password was either missing or empty.

Reply:
Updated.

Reply:
Updated.